frontend/src/icons/user.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function User({className}) {
+  return (
+    <svg className={className}  xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you make .svg files with these class names and store it in assets ? 
How many classNames can be in this case ?